main.js

```diff
@@ -21,4 +23,361 @@ hamImg.addEventListener('click', menu);
+  section.style.display = 'block';
+  document.querySelector('#main').style.filter = 'blur(5px)';
+  document.querySelector('#crossPop').addEventListener('click', () => {
+    section.style.display = 'none';
+
+    document.querySelector('#main').style.filter = 'none';
+  });
+}
+
+function loadDataPopFirst() {
+  popUpWinFirst.innerHTML = `
+     <div class="projectPreview">
+        <div class="head">
+          <div id= "name_container">
+            <h2>${projectInfo[0].title}</h2>
+          </div>
+          <img id ="crossPop" src="icons/cross.png" alt="cross">
+        </div>
+        <ul class="technologiesPreview">
+          ${getTechList(projectInfo[0].technologies)}
+        </ul>
+        <div class="project_elements">
+          <img class="projectImg" src="${projectInfo[0].imagePop}" alt="pop up image" />
+          <div class="details_container">
+            <div class="projectDescription">
+                ${projectInfo[0].description}  and ${projectInfo[0].moreText}
+            </div>
+            <div class="projectButtons">
+              <button class="cardButtons" href="${projectInfo[0].link}">
+                <span>See Live</span>
+                <img src="icons/seeLiveIcon.svg" alt="live" />
+              </button>
+              <button class="cardButtons" href="${projectInfo[0].github}">
+                <span>See Source</span>
+                <img src="icons/whiteGitHubIcon.png" alt="Source" />
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      `;
+
+  popUpOPenClose(popUpWinFirst);
+}
+
+document.querySelector('#popUpContFirstBtn').addEventListener('click', loadDataPopFirst);
+function loadDataPopCard() {
+  projectInfo.forEach((info) => {
+    popUpWinSecond.innerHTML = `
+      <div class="projectPreview">
+        <div class="head">
+          <div id= "name_container">
+            <h2>${info.title}</h2>
+          </div>
+          <img id ="crossPop" src="icons/cross.png" alt="cross">
+        </div>
+        <ul class="technologiesPreview">
+          ${getTechList(info.technologies)}
+        </ul>
+        <div class="project_elements">
+          <img class="projectImg" src="${info.imagePop}" alt="pop up image" />
+          <div class="details_container">
+            <div class="projectDescription">
+                ${info.description} and ${info.moreText}
+            </div>
+            <div class="projectButtons">
+              <button class="cardButtons" href="${info.link}">
+                <span>See Live</span>
+                <img src="icons/seeLiveIcon.svg" alt="live" />
+              </button>
+              <button class="cardButtons" href="${info.github}">
+                <span>See Source</span>
+                <img src="icons/whiteGitHubIcon.png" alt="Source" />
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      `;
+  });
+  popUpOPenClose(popUpWinSecond);
+}
+
```

Comment:
You have done a great job implementing some of the changes as requested in the previous review😉😉 but I think your popup still looks a bit different from the design, both on mobile and desktop.

popup (desktop)
<img width="905" alt="your___popuopdesktop" src="https://user-images.githubusercontent.com/64643995/217735228-4c425c7b-12ef-4602-b92f-6e527ae1bcac.png">

on the desktop view, your image is larger than the assigned space for the image container😉😉 you can see the right appearance in the[ image ](https://user-images.githubusercontent.com/64643995/217587106-59316775-45df-4f26-ba76-4abfa945f521.png)uploaded in the previous review or in the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tr7qidNAMlmCpUo8-0)😉😉😉😉😉

popup (mobile)
<img width="274" alt="popup_yoursmobile" src="https://user-images.githubusercontent.com/64643995/217736778-e50562cc-f153-4d05-b4b1-eea7f8c42d12.png">

on mobile, your texts are also overflowing, tho I can see there's a  bit improvement😉 kindly make comparison with the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tr7qidNAMlmCpUo8-0) or the[ image](https://user-images.githubusercontent.com/64643995/217587795-2bde8167-d6de-4fce-8e52-e9d0027271c7.png) in the previous review🤗🤗. don't forget it is a requirement that we stick to the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tr7qidNAMlmCpUo8-0) in this milestone of the project😉😉 kindly make the necessary adjustments🤗🤗